src/index.scss

```diff
@@ -5,6 +5,11 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 
 @link https://github.com/carbon-design-system/carbon/issues/2789
 */
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
 main {
   display: block;
 }
+$feature-flags: (
```

Comment:
The $feature-flags section has to come first in the scss file BEFORE importing the carbon-components styles.